app/services/ui-development.html

```diff
@@ -100,6 +100,19 @@ <h2 class="title my-4">Front-end development</h2>
         <p>
           We have extensive experience in working with:
         </p>
+
+        <div class="d-flex justify-content-between align-items-center mt-5">
+          <picture>
+            <img src="../images/services/react-logo.svg" alt="React Logo" title="React" class="img-fluid">
```

Comment:
Please make this **React.js**

Reply:
in the `alt` attribute please change *Logo* to **logo**

Reply:
Typo error is fixed